integration_tests/plugins/index.ts

```diff
@@ -23,5 +23,6 @@ export default (on: (string, Record) => void): void => {
       ]),
     stubLoginPage: auth.redirect,
     stubNotificationPreferencesGet: (arg) => notificationService.stubNotificationPreferencesGet(arg),
+    verifySnooze: () => notificationService.verifySnooze(),
```

Comment:
unnecessary line since
```
    ...notificationService,
```
exposes all the methods from notification service anyway

Reply:
Good spot